swift/json_object_array.swift

```diff
@@ -0,0 +1,21 @@
+//Script writes an array of objects to stdout
+import Foundation
+
+guard CommandLine.arguments.count > 1 else {
+    print("Usage: swift script.swift <arg1> [<arg2> ...]")
+    exit(1)
+}
+
+let args = Array(CommandLine.arguments.dropFirst())
+
+let myArray = args.map { arg -> [String: Any] in
+    return [arg.uppercased(): arg.count]
+}
+
+let jsonData = try JSONSerialization.data(withJSONObject: myArray, options: .prettyPrinted)
```

Comment:
```suggestion
let jsonData = try JSONSerialization.data(withJSONObject: myArray)
```

Let's remove the `options: .prettyPrinted` from other places it's used, too

Reply:
Make sure to get all other instances of `.prettyPrinted`